javascript-operators-and-expressions/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en-us">
+
+<head>
+
```

Comment:
Let's get rid of some of these empty lines. It is a matter of preference but I wouldn't have any more than one empty line above or below the code.